main.go

```diff
@@ -240,6 +236,13 @@ func main() {
 		log.Fatal(err)
 	}
 
+	var processFunc func(warehouse.Warehouse, *fullstory.Client, []fullstory.ExportMeta) (int, error)
```

Comment:
Nit: this could be expressed more cleanly as:
```
processFunc := ProcessFilesIndividually
if conf.GroupFilesByDay {
    processFunc = ProcessFilesByDay
}
```
since processing individually is the default